### PR TITLE
IDEM-150

### PR DESCRIPTION
**Pull request by JasminFarhi**
Convert to HTML to React
Deploy to DO app
Set up CI/CD/husky

### CODE REVIEW

File: .github/workflows/eas-build.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        with:
+          commit_message: Bump version number
+
+      - name: Notify Slack
+        if: ${{ always() && github.ref == 'refs/heads/main' }}
+        uses: ./.github/actions
+        with:
+          slack-channel: "idem"
+          slack-bot-token: ${{ secrets.SLACK_NOTIFICATIONS_BOT_TOKEN }}
+          outcome: ${{ steps.bump.outcome }}
```
Comment (Mitchell139) [inline .github/workflows/eas-build.yml:66]: I don't think you want this. This is for building react native apps (right)

File: app/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+## Learn More
+
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (Mitchell139) [inline app/README.md:46]: This is not a react app

File: .husky/pre-push [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+echo "Husky pre-push"
+# yarn build
```
Comment (Mitchell139) [inline .husky/pre-push:5]: Is this commented out and if so, shouldn't it not be commented out?

File: .husky/pre-commit [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# yarn format-staged && yarn lint
```
Comment (Mitchell139) [inline .husky/pre-commit:4]: Same

**Review by Mitchell139 — CHANGES_REQUESTED** [review]

File: .github/actions/action.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Action prerequisite - Slack channel installation instructions
+# 1) Open the target DLTx slack channel info modal
+# 2) On the Integration tab click 'Add apps'
+# 3) Select 'GitHub Action' app already in the workspace
+# 4) Verify there is a message from the app in the channel
+name: "Notify Slack"
+description: "Send a notification to Slack indicating the workflow status"
+inputs:
+  slack-channel: # id of input
+    description: "Slack channel name"
```
Comment (bitcoinbrisbane) [inline .github/actions/action.yml:10]: Should this be the acutally channel?

File: app/public/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (bitcoinbrisbane) [inline app/public/index.html:10]: Lets change this to something better

File: app/src/App.test.tsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (bitcoinbrisbane) [inline app/src/App.test.tsx:9]: Needs to be linited to our standardard

File: app/src/App.tsx [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
```
Comment (bitcoinbrisbane) [inline app/src/App.tsx:26]: Needs to be linted and an arrow function

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
- Create arrow functions
- Remove the tests or get one working
- Lint and add our preittier

**Pull request closed without merging**